app/interactors/update_match_score.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class UpdateMatchScore
+  include Interactor
+
+  def call
+    match_score = context.match_score
+    match_score_params = context.match_score_params
+    if match_score.update(match_score_params)
+      match_score.match.state = 'in_progress'
```

Comment:
Why are you setting the state to in_progress all the time?
You probably want to call your evaluate_status method instead...

Reply:
As discussed we will take care of updating the match state in a separate ticket/pr, therefore remove this line pls.